### PR TITLE
PDE Updates

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR adds new content relegated to PDE enhancements.

## Review Changes

- [Updated PDE Index Page](https://deploy-preview-1293--docs-spectrocloud.netlify.app/devx#palettecli)
CleanShot 2023-05-15 at 16 48 52

- [Updated the PDE Management Section](https://deploy-preview-1293--docs-spectrocloud.netlify.app/devx/manage-dev-engine)
CleanShot 2023-05-15 at 16 50 25

- [Added CLI Install Page](https://deploy-preview-1293--docs-spectrocloud.netlify.app/devx/manage-dev-engine/install-palette-cli)
- Moved Resource Quotas and Registries page to the PDE Management section.

- Updated Create App Profile How-to to include the new app profile layer information boxes.
CleanShot 2023-05-16 at 07 38 02

- PDE Index page updated to display the centralized resource view available from the dashboard.
CleanShot 2023-05-16 at 10 06 07

- Mentioned replicas in the Create an App Profile Guide.

- [Added section for Palette CLI](https://deploy-preview-1293--docs-spectrocloud.netlify.app/palette-cli)

💻 [Preview URL](https://deploy-preview-1293--docs-spectrocloud.netlify.app/devx)

🎫 [PPD-1219](https://spectrocloud.atlassian.net/browse/PPD-1219)
🎫 [PPD-1220](https://spectrocloud.atlassian.net/browse/PPD-1220)
🎫 [PPD-1218](https://spectrocloud.atlassian.net/browse/PPD-1218)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1293#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-4/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04.5-devx/09-manage-dev-engine/30-sso.md
Comment (ritawatson) [inline content/docs/04.5-devx/09-manage-dev-engine/30-sso.md:16]:
```suggestion
Palette supports the ability to use Single Sign-On (SSO) and third-party Social Sign-In Providers, such as Google and GitHub. Use the following steps to either enable or disable the feature. 
```

File: content/docs/04.5-devx/09-manage-dev-engine/30-sso.md
Comment (ritawatson) [inline content/docs/04.5-devx/09-manage-dev-engine/30-sso.md:42]:
```suggestion
2. Navigate to the left **Main Menu**, select **Tenant Settings**, and select **SSO**.
```

File: content/docs/04.5-devx/09-manage-dev-engine/30-sso.md
Comment (ritawatson) [inline content/docs/04.5-devx/09-manage-dev-engine/30-sso.md:39]: I've seen "Tenant Admin" and "tenant admin" in our docs. I'll add it to the Agenda to discuss.

File: content/docs/04.5-devx/09-manage-dev-engine/30-sso.md
Comment (ritawatson) [inline content/docs/04.5-devx/09-manage-dev-engine/30-sso.md:51]:
```suggestion
4. Select one of the supported Social Sign-In providers, and confirm your change.
```

File: content/docs/04.5-devx/09-manage-dev-engine/30-sso.md
Comment (ritawatson) [inline content/docs/04.5-devx/09-manage-dev-engine/30-sso.md:54]: Is this a necessary step? Maybe it's needed in order to validate? If so, you might add that bit of information.

File: content/docs/04.5-devx/09-manage-dev-engine/30-sso.md
Comment (ritawatson) [inline content/docs/04.5-devx/09-manage-dev-engine/30-sso.md:49]: Add a break here.

File: content/docs/04.5-devx/09-manage-dev-engine/30-sso.md
Comment (ritawatson) [inline content/docs/04.5-devx/09-manage-dev-engine/30-sso.md:63]: Add a break here.

File: content/docs/04.5-devx/09-manage-dev-engine/30-sso.md
Comment (ritawatson) [inline content/docs/04.5-devx/09-manage-dev-engine/30-sso.md:67]:
```suggestion
Palette provides the flexibility to disable SSO to restrict this capability. Use the following steps to disable SSO for Palette.
```

File: content/docs/04.5-devx/09-manage-dev-engine/30-sso.md
Comment (ritawatson) [inline content/docs/04.5-devx/09-manage-dev-engine/30-sso.md:81]:
```suggestion
2. Navigate to the left **Main Menu**, select **Tenant Settings**, and select **SSO**.
```

File: content/docs/04.5-devx/09-manage-dev-engine/30-sso.md
Comment (ritawatson) [inline content/docs/04.5-devx/09-manage-dev-engine/30-sso.md:45]:
```suggestion
3. Next, click the **Auth Providers** tab and toggle the **Enable Provider Login** button on.
```

File: content/docs/04.5-devx/09-manage-dev-engine/30-sso.md
Comment (ritawatson) [inline content/docs/04.5-devx/09-manage-dev-engine/30-sso.md:84]:
```suggestion
3. Next, click the **Auth Providers** tab and toggle the **Enable Provider Login** button off.
```

File: content/docs/14-troubleshooting/10-palette-dev-engine.md
Comment (ritawatson) [inline content/docs/14-troubleshooting/10-palette-dev-engine.md:16]:
```suggestion
Use the following content to help you troubleshoot issues you may encounter when using Palette Dev Engine (PDE).
```

**Review by ritawatson — APPROVED** [review]
You've been busy with all these 3.4 fixes! Thank you Karl. 
Some fixes and comments. When those are in, LGTM.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1293#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1293#pullrequestreview-1429397572" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-4/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1293#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1293#pullrequestreview-1429397572" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-4/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/00-release-notes.md
Comment (tiwarisumit3) [inline content/docs/00-release-notes.md:135]: This means that exposed service URLs automatically default to valid certificate-based dynamically generated DNS URLs.

File: content/docs/04.5-devx/09-manage-dev-engine/10-install-palette-cli.md
Comment (tiwarisumit3) [inline content/docs/04.5-devx/09-manage-dev-engine/10-install-palette-cli.md:110]: The publicly available one will have only pde sub-command available. Let us document that here.

**Review by tiwarisumit3 — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1293#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1293#pullrequestreview-1429397572" title="Approved">ritawatson</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1293#pullrequestreview-1429534010" title="Approved">tiwarisumit3</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-4/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/00-release-notes.md
Comment (karl-cardenas-coding) [inline content/docs/00-release-notes.md:135]:
```suggestion
- Container applications that expose a service now automatically receive ingress support with HTTPS support out-of-the-box. This means exposed service URLs automatically receive dynamic SSL certificates used for HTTPS.
```

File: content/docs/04.5-devx/09-manage-dev-engine/30-sso.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/09-manage-dev-engine/30-sso.md:54]:
```suggestion
```

File: content/docs/04.5-devx/05-app-profile/02-container-deployment.md
Comment (karl-cardenas-coding) [reply content/docs/04.5-devx/05-app-profile/02-container-deployment.md:74]: Fixed

File: content/docs/04.5-devx/09-manage-dev-engine/10-install-palette-cli.md
Comment (karl-cardenas-coding) [reply content/docs/04.5-devx/09-manage-dev-engine/10-install-palette-cli.md:110]: I'm just gonna replace this with the version command @tiwarisumit3 . That way we get away from this issue,

File: content/docs/12.5-palette-cli/20-commands.md
Comment (ritawatson) [inline content/docs/12.5-palette-cli/20-commands.md:16]:
```suggestion
You start the Palette CLI with a single command, `palette`. The CLI accepts various subcommands such as `pde`, `help`, and more. The Palette CLI will return a non-zero exit status during error scenarios. You can use the CLI flags `-h` and `--help` to learn more about each subcommand.
```

File: content/docs/12.5-palette-cli/20-commands.md
Comment (ritawatson) [inline content/docs/12.5-palette-cli/20-commands.md:18]:
```suggestion
The complete list of subcommands is:
```

File: content/docs/12.5-palette-cli/20-commands.md
Comment (ritawatson) [inline content/docs/12.5-palette-cli/20-commands.md:38]:
```suggestion
| `-c`       | `--config`      | Config file location.                                                     | string  |
```

File: content/docs/12.5-palette-cli/20-commands.md
Comment (ritawatson) [inline content/docs/12.5-palette-cli/20-commands.md:54]:
```suggestion
  * `login`         -   Log in to Spectro Cloud Palette. 
```

File: content/docs/12.5-palette-cli/20-commands.md
Comment (ritawatson) [inline content/docs/12.5-palette-cli/20-commands.md:85]:
```suggestion
  | `--api-key`             | The Palette API key.                                  |  string    | Yes |
```

File: content/docs/12.5-palette-cli/20-commands.md
Comment (ritawatson) [inline content/docs/12.5-palette-cli/20-commands.md:86]:
```suggestion
  | `--cluster-group-name`  | The Palette Cluster Group name.                        |  string    |   No       |
```

File: content/docs/12.5-palette-cli/20-commands.md
Comment (ritawatson) [inline content/docs/12.5-palette-cli/20-commands.md:87]:
```suggestion
  | `--cluster-group-scope` | Palette Cluster Group scope.                           |   string   | No |
```

File: content/docs/12.5-palette-cli/20-commands.md
Comment (ritawatson) [inline content/docs/12.5-palette-cli/20-commands.md:88]:
```suggestion
  | `--console-url`         | The Palette login URL.                              |  string    | Yes |
```

File: content/docs/12.5-palette-cli/20-commands.md
Comment (ritawatson) [inline content/docs/12.5-palette-cli/20-commands.md:109]:
```suggestion
  * `list`   -      List Palette projects.
```

File: content/docs/12.5-palette-cli/10-install-palette-cli.md
Comment (ritawatson) [inline content/docs/12.5-palette-cli/10-install-palette-cli.md:30]: I guess we'll be adding the URL to the Downloads page in a few days?

File: content/docs/12.5-palette-cli/10-install-palette-cli.md
Comment (ritawatson) [inline content/docs/12.5-palette-cli/10-install-palette-cli.md:45]: The second sentence is awkward to me. Are we replacing an old file name with a new file name? Or are filling in a placeholder value?

File: content/docs/12.5-palette-cli/10-install-palette-cli.md
Comment (ritawatson) [inline content/docs/12.5-palette-cli/10-install-palette-cli.md:57]:
```suggestion
Move the binary to a folder on your system. If you want to access the Palette CLI from the command line, ensure you place the binary in a folder that is part of your system `PATH`. Check out the [Add Tool Locations to the PATH Environment Variable](https://learn.microsoft.com/en-us/previous-versions/office/developer/sharepoint-2010/ee537574(v=office.14)) for detailed steps.
```

File: content/docs/12.5-palette-cli/10-install-palette-cli.md
Comment (ritawatson) [inline content/docs/12.5-palette-cli/10-install-palette-cli.md:59]:
```suggestion
Open the Command Prompt and move the file to a target folder that is part of your system `PATH`, as shown in the following command example. 
```

File: content/docs/12.5-palette-cli/10-install-palette-cli.md
Comment (ritawatson) [inline content/docs/12.5-palette-cli/10-install-palette-cli.md:78]:
```suggestion
5. If you will use Palette Dev Engine (PDE) complete this step. Otherwise you can skip to [Validation](/palette-cli/install-palette-cli/#validation). Log in to Palette by using the `login` command. Replace `<YOUR-API-KEY>` with your Palette API key. If you are using a Palette self-hosted instance, replace the `--console-url` with your custom Palette URL.
```

File: content/docs/12.5-palette-cli/10-install-palette-cli.md
Comment (ritawatson) [inline content/docs/12.5-palette-cli/10-install-palette-cli.md:88]:
```suggestion
Verify the Palette CLI is part of your system path by issuing the Palette CLI `version` command.
```

Comment (ritawatson) [reply content/docs/12.5-palette-cli/10-install-palette-cli.md:88]: Did you mean `version`? 
The `version` command is not listed on the Command Reference page.

File: content/docs/12.5-palette-cli/10-install-palette-cli.md
Comment (ritawatson) [inline content/docs/12.5-palette-cli/10-install-palette-cli.md:104]: Add some breaks after this section to separate it from Previous/Next navigators.

Comment (ritawatson) [inline content/docs/12.5-palette-cli/10-install-palette-cli.md:104]:
```suggestion
Start exploring the Palette CLI by using the `--help` command with the various commands. The Palette CLI will continue to receive more functionality, so you will want to keep it updated by downloading the newest version and replacing the current binary.
```

**Review by ritawatson — APPROVED** [review]
LGTM with comments.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1293#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1293#pullrequestreview-1431099871" title="Approved">ritawatson</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1293#pullrequestreview-1429534010" title="Approved">tiwarisumit3</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-4/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/12.5-palette-cli/10-install-palette-cli.md
Comment (karl-cardenas-coding) [inline content/docs/12.5-palette-cli/10-install-palette-cli.md:45]:
```suggestion
Use the following command to move the binary to the **/usr/local/bin** folder.
```

File: content/docs/04.5-devx/05-app-profile/02-container-deployment.md
Comment (karl-cardenas-coding) [reply content/docs/04.5-devx/05-app-profile/02-container-deployment.md:1]: @ritawatson, we should only log in as a tenant admin when warranted.

File: content/docs/12.5-palette-cli/20-commands.md
Comment (karl-cardenas-coding) [inline content/docs/12.5-palette-cli/20-commands.md:88]:
```suggestion
  | `--console-url`         | The Palette login URL. The default value is `https://console.spectrocloud.com`                             |  string    | No |
```

**Pull request merged**
